xapian-letor/tests/api_letor.cc

```diff
@@ -194,19 +194,20 @@ DEFINE_TESTCASE(listmle_ranker, generated)
     string data_directory = test_driver::get_srcdir() + "/testdata/";
     string query = data_directory + "query.txt";
     string qrel = data_directory + "qrel.txt";
-    Xapian::prepare_training_file(db_path, query, qrel, 10,
-				  "listmle_ranker_training.txt");
+    string training = test_driver::get_srcdir()
+	    + "listmle_ranker_training.txt";
```

Comment:
Xapian convention is to break a line after an operator, not before (so `+` should be at the end of the first line, not the start of the second).